modules/migration-mtc-workflow.adoc

```diff
@@ -56,10 +56,12 @@ image::migration-PV-move.png[]
 
 . Run the migration plan, with one of the following options:
 
-* *Stage* (optional) copies data to the target cluster without stopping the application.
+* *Stage* copies data to the target cluster without stopping the application.
 +
-Staging can be run multiple times so that most of the data is copied to the target before migration. This minimizes the duration of the migration and the application downtime.
+A stage migration can be run multiple times so that most of the data is copied to the target before migration. This reduces the duration of the cutover migration.
```

Comment:
With the last sentence, I would say "This plan" or "This approach" to make it clear that this action from the previous sentence reduces the duration.

Reply:
I will rewrite. It's not a plan or an approach but I can think of an alternative.